### PR TITLE
Fleshing out initial docs

### PR DESCRIPTION
**Pull request by foresmac**
@adambain-vokal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,66 +2,88 @@
...
+1. [Get a list of Cards](#get-a-list-of-cards)
+1. [Get a Card's details](#get-a-cards-details)
+1. [Get Collection](#get-collection)
+1. [Add a Card to Collection](#add-a-card-to-collection)
+1. [Remove a Card from Collection](#remove-a-card-from-collection)
+1. [Create a Deck](#create-a-deck)
+1. [Get a list of Decks](#get-a-list-of-decks)
+1. [Get details of a Deck](#get-details-of-a-deck)
+1. [Update a Deck](#update-a-deck)
+1. [Delete a Deck](#delete-a-deck)
```
Comment (foresmac) [inline README.md:18]: Do you see anything missing here?

Comment (adambain-vokal) [inline README.md:18]: No

**Comment by adambain-vokal** [comment]
LGTM

**Comment by foresmac** [comment]
thanks

**Pull request merged**